src/formElementPlugins/ApplicationViewWrapper.tsx

```diff
@@ -250,32 +245,19 @@ const getDefaultIndex = (defaultOption: string | number, options: string[]) => {
   } else return options.indexOf(defaultOption)
 }
 
-export const extractDynamicExpressions = (
-  fields: string[],
-  parameters: ElementPluginParameters
-) => {
-  const expressionObject: ElementPluginParameters = {}
-  fields.forEach((field) => {
-    expressionObject[field] = parameters[field]
-  })
-  return expressionObject
-}
-
-export const evaluateDynamicParameters = async (
-  dynamicExpressions: ElementPluginParameters,
-  evaluatorParameters: EvaluatorParameters
+export const buildParameters = (
+  parameters: ElementPluginParameters,
+  parameterLoadingValues: any
 ) => {
-  if (!dynamicExpressions) return {}
-  const fields = Object.keys(dynamicExpressions)
-  const expressions = Object.values(dynamicExpressions).map(
-    (expression: ElementPluginParameterValue) => evaluateExpression(expression, evaluatorParameters)
-  )
-  const evaluatedExpressions: any = await Promise.all(expressions)
-  const evaluatedParameters: ElementPluginParameters = {}
-  for (let i = 0; i < fields.length; i++) {
-    evaluatedParameters[fields[i]] = evaluatedExpressions[i]
+  const simpleParameters: any = {}
+  const parameterExpressions: any = {}
+  for (const [key, value] of Object.entries(parameters)) {
+    if (value instanceof Object && !Array.isArray(value)) {
+      parameterExpressions[key] = value
+      simpleParameters[key] = parameterLoadingValues?.[key] ?? 'Loading...'
```

Comment:
wonder if we need to localise loading

Reply:
Yeah, we probably do, although I wasn't sure how to handle localisations that are in the config.json file. (And I didn't think there was much point moving this to constants if the config values weren't also.)

Reply:
But this is not in config.json ?

Reply:
`parameterLoadingValues` are in pluginConfig.json, this is just a fallback. What I was meaning is that there isn't a lot of point localising this string unless we can also localise the ones in the .json file.